From a78f785e776c674f8151a53243245890040be5e6 Mon Sep 17 00:00:00 2001
From: Alexander Stein <alexander.stein@ew.tq-group.com>
Date: Fri, 17 Feb 2023 10:52:20 +0100
Subject: [PATCH] media: dt-bindings: media: i2c: Add imx327 version to
 IMX327 bindings

Should be upstream commit 783ecfe5556a

The imx290 driver can be used for both imx290 and imx327 as they have a
similar register set and configuration. imx327 lacks 8 lanes LVDS and
120 FPS support.

Signed-off-by: Alexander Stein <alexander.stein@ew.tq-group.com>
Acked-by: Krzysztof Kozlowski <krzysztof.kozlowski@linaro.org>
Signed-off-by: Sakari Ailus <sakari.ailus@linux.intel.com>
---
 Documentation/devicetree/bindings/media/i2c/sony,imx290.yaml | 1 +
 1 file changed, 1 insertion(+)

--- a/Documentation/devicetree/bindings/media/i2c/sony,imx290.yaml
+++ b/Documentation/devicetree/bindings/media/i2c/sony,imx290.yaml
@@ -29,6 +29,7 @@ properties:
       - enum:
           - sony,imx290lqr # Colour
           - sony,imx290llr # Monochrome
+          - sony,imx327lqr # Colour
       - const: sony,imx290
         deprecated: true
 
